generative_data_prep/utils/constants.py

```diff
@@ -93,3 +93,21 @@ class FileExtension(str, BaseEnum):
 
 
 CATEGORY_JSON_KEY = "category"
+
```

Comment:
Can you add a `# TODO: <description>` here so that we can revert it later? You can mention the BTD ticket in the description.